packages/server/tests/acceptance/rpc.spec.ts

```diff
@@ -1149,6 +1150,49 @@ describe('@api RPC Server Acceptance Tests', function () {
             });
         });
 
+        // Test state changes with getStorageAt
+        describe('eth_getStorageAt', () => {
+            let storageContract, contractId, evmAddress;
+            const STORAGE_CONTRACT_UPDATE = "0x2de4e884";
+
+            before(async () => {
+                storageContract = await servicesNode.deployContract(storageContractJson);
+                // Wait for creation to propagate
+                await mirrorNode.get(`/contracts/${storageContract.contractId}`);
+
+                contractId = storageContract.contractId;
+                evmAddress = `0x${storageContract.contractId.toSolidityAddress()}`;
+            });
+
+            it('@release should execute "eth_getStorageAt" request to get state changes', async function () {
+                const gasPrice = await relay.gasPrice();
+                const transaction = {
+                    value: 0,
+                    gasLimit: 50000,
+                    chainId: Number(CHAIN_ID),
+                    to: evmAddress,
+                    nonce: await relay.getAccountNonce(accounts[1].address),
+                    gasPrice: gasPrice,
+                    data: STORAGE_CONTRACT_UPDATE,
+                    maxPriorityFeePerGas: gasPrice,
+                    maxFeePerGas: gasPrice,
+                    type: 2
+                };
+
+                const signedTx = await accounts[1].wallet.signTransaction(transaction);
+                const transactionHash = await relay.call('eth_sendRawTransaction', [signedTx]);
+
+                const getStoragAtParams = {
```

Comment:
`eth_getStorageAt` params should be an array of strings where position 0 is an address, 1 is the slot in hex and 3 is the block number in hex or tag